packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -16,11 +16,31 @@ interface AutocompleteProps {
    */
   readonly value: Option | undefined;
 
+  /**
+   * Hint text that goes above the value once the form is filled out.
+   */
+  readonly menuSize?: "medium";
+
   /**
    * Hint text that goes above the value once the form is filled out.
    */
   readonly placeholder: string;
 
+  /**
+   * Turn on separators between result elements.
+   */
+  readonly separators?: boolean;
+
+  /**
+   * Turn off checkmarks on selected result elements.
+   */
+  readonly checkmarks?: boolean;
+
+  /**
+   * Turn on browser autocomplete for the input.
+   */
+  readonly browserAutocomplete?: string;
```

Comment:
I don't think this should ever be on?

Reply:
can we just remove the option to turn it on altogether?

Reply:
If there's no reason to enable it the I think we don't need it as a prop.